[2023-06-15T10:39:01.433+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T10:38:00+00:00 [queued]>
[2023-06-15T10:39:01.439+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T10:38:00+00:00 [queued]>
[2023-06-15T10:39:01.439+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T10:39:01.440+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-15T10:39:01.440+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T10:39:01.447+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): add_entry_task> on 2023-06-15 10:38:00+00:00
[2023-06-15T10:39:01.451+0000] {standard_task_runner.py:55} INFO - Started process 836 to run task
[2023-06-15T10:39:01.454+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'add_entry_dag', 'add_entry_task', 'scheduled__2023-06-15T10:38:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/postgres_dag.py', '--cfg-path', '/tmp/tmp7t3xrjq6']
[2023-06-15T10:39:01.456+0000] {standard_task_runner.py:83} INFO - Job 45: Subtask add_entry_task
[2023-06-15T10:39:01.504+0000] {task_command.py:389} INFO - Running <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T10:38:00+00:00 [running]> on host ff7147a0d5a7
[2023-06-15T10:39:01.549+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=add_entry_dag
AIRFLOW_CTX_TASK_ID=add_entry_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-15T10:38:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-15T10:38:00+00:00
[2023-06-15T10:39:01.558+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-15T10:39:01.563+0000] {sql.py:364} INFO - Running statement: INSERT INTO assignment_table (execution_time) VALUES ('2023-06-15 10:39:01.550912'), parameters: None
[2023-06-15T10:39:01.564+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-15T10:39:01.565+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-15T10:39:01.573+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=add_entry_dag, task_id=add_entry_task, execution_date=20230615T103800, start_date=20230615T103901, end_date=20230615T103901
[2023-06-15T10:39:01.626+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-15T10:39:01.647+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
